<ui version="4.0" >
 <class>ResolveWordPage</class>
 <widget class="QWidget" name="ResolveWordPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lbWordDesc" >
     <property name="text" >
      <string>Wort:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leWord" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lbTypeDesc" >
     <property name="text" >
      <string>Typ:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="cbType" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lbSampa" >
     <property name="text" >
      <string>Sam-Pa:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="leSampa" />
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QTabWidget" name="twShower" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Ähnliche Wörter</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTableWidget" name="twSuggestions" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>94</height>
          </size>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text" >
           <string>Wort</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Betonung</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Typ</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Erkennungsrate</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Beispiele</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="lbSample1Desc" >
         <property name="maximumSize" >
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>1:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lbSample2Desc" >
         <property name="maximumSize" >
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>2:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbReGuess" >
           <property name="text" >
            <string>Neu Raten</string>
           </property>
           <property name="icon" >
            <iconset resource="../action_icons.qrc" >:/images/icons/view-refresh.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="leExample1" />
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="leExample2" />
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="lbError1" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="lbError2" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../action_icons.qrc" />
 </resources>
 <connections/>
</ui>
